master/docs/manual/mydashboard.html

```diff
@@ -1 +0,0 @@
-../../../smokes/templates/mydashboard.html
```

Comment:
It does.. and i have them enabled for my logon. I am not sure what happened here.